port/resource_port_blueprint.go

```diff
@@ -306,6 +317,20 @@ func readBlueprint(ctx context.Context, d *schema.ResourceData, m interface{}) d
 	return diags
 }
 
+func isDeprecatedDefaultExists(identifier string, d *schema.ResourceData, propertyName string) bool {
```

Comment:
If it's deprecated default, you don't need the property name, just use the hardcoded "default"